active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,349 @@
+| DefaultPipStyle and SelectedPipStyle | Gives the developer the option to customize the style by changing the appearance of the pips. | Based on default and selected pip theme resources and the PipsPagerButtonBaseStyle defined in theme resources.
+| NextButtonVisibility | Sets the visibility of the button that navigates to the next item. | Collapsed |
+| PreviousButtonVisibility | Sets the visibility of the button that navigates to the previous item. | Collapsed |
+| SelectedPageIndex | The 0 based index that is currently selected. It will default to the first index. | 0
+| MaxVisiblePips | The maximum number of pips that will appear in the control at once. If the NumberOfPages is greater than this value, the UI will scroll so that the selected pip is centered. | 5
+| Orientation | The orientation of the control. Can be either Vertical or Horizontal. | Horizontal
+| PipsPagerSelectedIndexChangedEvent | Event that is fired after the user selects a pip or the directional buttons. This event will return the index number of the page that the user selected. | N/A
+
+## PipsPager Theme Resources
+
+You can modify the look of a PipsPager by specifying Xaml resources in your app.
+For more info, see the
+[lightweight styling guide](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-styles#lightweight-styling).
+
+| Resource key | Description | Type
+|:-:|:--| :-- |
+| PipsPagerButtonWidth | Sets the bounding box width for each pip | Double |
+| PipsPagerButtonHeight | Sets the bounding box height for each pip | Double |
+| PipsPagerButtonBorderThickness | Sets the border thickness for each pip | Thickness |
+| PipsPagerNavigationButtonBorderThickness | Sets the border thickness for each navigational button | Thickness |
+| PipsPagerSelectedGlyph | Sets the pip glyph when selected in MDL2 icon set | String |
+| PipsPagerDefaultGlyph | Sets the default pip glyph in MDL2 icon set | String |
+| PipsPagerNavigationButtonHeight | Sets the height for each navigational button | Double |
+| PipsPagerNavigationWidth | Sets the width for each navigational button | Double |
+| PipsPagerNavigationButtonFontSize | Sets the font size for each navigational button | Double |
+| PipsPagerSelectedGlyphFontSize | Sets the size in pixels of the pip glyph when selected | Double |
+| PipsPagerDefaultGlyphFontSize | Sets the size in pixels of the pip glyph at rest| Double |
+
+## PipsPagerButtonVisibility enum
+
+Defines constants that specify how a the buttons of a PipsPager are to be displayed.
+
+| Value | Description |
+|:---:|:---|
+| Visible | The button is visible and enabled, except it is hidden at the extents. For example, when the current page is the first page, the previous button is hidden. |
+| VisibleOnPointerOver | The behavior is the same as Visible *except* that the button is only Visible when the user is hovering over the paging UI with their cursor. |
```

Comment:
Is there a way to get them to be visible on touch systems? Or is the idea that on touch, you just touch the pip you want, so the arrows aren't necessary? (Except as a visual cue that there are more items.)

Reply:
The idea is that on touch, you would touch the pip you want or use the touch interaction on the collection to navigate between the pages. 

I'll take a note to edit this description to reflect that the navigational button is also visible on keyboard focus.

Reply:
Recommend: yes please on the note